src/components/tooltip/Tooltip.tsx

```diff
@@ -205,7 +208,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
             constrainToWindow,
             isDisabled,
             isTabbable = true,
-            position,
+            position = TooltipPosition.TOP_CENTER,
```

Comment:
Would this already be defaulted by the defaultProps on L109?

Reply:
I think it's because we do `Partial<DefaultTooltipProps>` in the Props definition (though we do set it in the `defaultProps`. the `tetherPosition` constant complains it could be undefined. 🤔

Reply:
Maybe if you change `defaultProps` type to `TooltipProps`?

Reply:
I have it set to `Partial` because none of those are "required" but maybe that isn't getting us anything different than if I put them back in `Tooltips` and made them optional?